qa/TL1_tensorflow_dataset/test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -e
```

Comment:
Why not TL0?

Reply:
It uses multi gpu runner. For now it's only this one simple test, but in the future I will add full multi gpu test here.